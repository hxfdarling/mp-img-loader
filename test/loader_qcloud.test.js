/* eslint-disable
  prefer-destructuring,
*/
import webpack from '@webpack-contrib/test-utils';

describe('Loader', () => {
  test('Defaults', async () => {
    const config = {
      loader: {
        test: /\.png$/,
        options: {
          mode: 'qcloud',
          qcloud: {},
        },
      },
    };

    const stats = await webpack('qcloud.js', config);
    const { source } = stats.toJson().modules[0];

    expect(source).toMatchSnapshot();
  });
});
