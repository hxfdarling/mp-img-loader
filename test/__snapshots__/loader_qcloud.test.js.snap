// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Defaults 1`] = `"qcloud  /Users/liuhua/github/mp-img-loader/test/fixtures/file.png"`;
